lib/vdc/DCMPAS.cpp

```diff
@@ -1304,10 +1347,10 @@ bool DCMPAS::_HasVertical(
 
 		itr = find(varnames.begin(), varnames.end(), s);
 		if (itr == varnames.end()) {
-			return(false);
+			return(true);
 		}
 	} 
-	return(true);
+	return(false);
```

Comment:
Is this logic correct? If the iterator equals to the end, it usually means not finding that element.

Reply:
Good catch! Thanks!